packages/office-ui-fabric-react/src/components/Slider/examples/Slider.Basic.Example.tsx

```diff
@@ -29,12 +45,24 @@ export class SliderBasicExample extends React.Component<{}, ISliderBasicExampleS
           min={0}
           max={50}
           step={10}
+          snapToStep
+          showValue={true}
+          onChange={(value: number) => console.log(value)}
+          defaultValue={20}
+        />
+
+        <Slider
+          label="Marks example"
+          min={0}
+          max={100}
+          step={10}
           defaultValue={20}
           showValue={true}
           onChange={(value: number) => console.log(value)}
-          snapToStep
```

Comment:
Bring back the snap example.

Reply:
not quite sure what you mean isnt the snaps exsample already there?

Reply:
Aren't you modifying it with marks?